src/ui/common/src/components/workflows/nodes/CheckOperatorNode.tsx

```diff
@@ -12,13 +25,131 @@ type Props = {
 export const checkOperatorNodeIcon = faMagnifyingGlass;
 
 const CheckOperatorNode: React.FC<Props> = ({ data, isConnectable }) => {
+  const defaultLabel = 'Check';
+  const label = data.label ? data.label : defaultLabel;
+  const result = data.result;
+  const currentNode = useSelector(
+    (state: RootState) => state.nodeSelectionReducer.selected
+  );
+  const workflowState = useSelector(
+    (state: RootState) => state.workflowReducer
+  );
+  const selected = currentNode.id === data.nodeId;
+  let execState: ExecState;
+  if (data.nodeType === ReactflowNodeType.Operator) {
```

Comment:
we don't need this check since the node here is an Operator node.
Can simplify this down.

Reply:
Code here is just copied over from the Node.tsx file :)